src/control/lib/hardware/sysfs/provider.go

```diff
@@ -35,6 +40,34 @@ func isNetwork(subsystem string) bool {
 	return false
 }
 
+func isNetvscDevice(path string, subsystem string) bool {
+	if subsystem != netvscSubsystem {
+		return false
+	}
+
+	file, err := os.Open(filepath.Join(path, "device", "uevent"))
+	if err != nil {
+		return false
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
```

Comment:
this looks like the kind of thing we should have utility function for, we have some utility functions for key/value access but it might not be a drop-in replacement for this block at the moment, might need some "massaging"

Reply:
I am not specifically requesting changes on this in this PR, just an optional suggestion.

Reply:
- [ ] Try to use utils key/value facitlities

Reply:
Hello @tanabarr ,

I was not able to find with utility functions you are referring.
Please, could you tell me where I could find them.
Thanks in advance.

Reply:
`src/control/common/env_utils.go:FindEnvValue()`

Reply:
Thanks for the pointers.
PR updated using these facilities.